phpunit/block-supports/aria-label-test.php

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
This unit test is identical to the test in the core backport PR:
https://github.com/WordPress/wordpress-develop/pull/8274/files#diff-0f32f800b7e03fe08c378308dfb322156ea606714e2f39d095971d02add72a93

On the other hand, the following unit test cannot be included in this PR because it will not pass unless the `aria-label` attribute is allowed by default in the `get_block_wrapper_attributes` function:
https://github.com/WordPress/wordpress-develop/pull/8274/files#diff-699d565e390f7fb85d4d1c7ade8706b63fe2cec31043e8e18117961ca3f7083d